tests/test_parse_failure_logging.py

```
@@ -0,0 +1,191 @@
+        with tempfile.TemporaryDirectory() as tmp:
+            ws_root = _seed_tabs_with_drifted_bubble(tmp)
+            global_db = os.path.join(tmp, "globalStorage", "state.vscdb")
+            with closing(sqlite3.connect(global_db)) as conn:
+                conn.execute(
+                    "INSERT INTO cursorDiskKV ([key], value) VALUES (?, ?)",
+                    ("bubbleId:cmp-ok:b-json", "{not valid json"),
+                )
+                conn.commit()
+            with self.assertLogs("services.workspace_tabs", level="WARNING") as cm:
+                with app.test_request_context("/api/workspaces/global/tabs"):
+                    payload, status = assemble_workspace_tabs("global", ws_root, rules=[])
```

Ruff (0.15.14)

[warning] Unpacked variable `payload` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)